tophat/jenkins-timeline-plugin
# Dashboard header

## Description
Adds a basic header with some utility buttons and status info to the timeline.

## DevQA
### DevQA Prep
Run `make build_all` to generate a new `hpi`, install it on your Jenkins instance. You can use the pipeline sample that is in `devResources` if you don't have one already.

### DevQA Steps
- Start the pipeline build
- Open up the timeline
- The header should display the start time, the running time, the status and the longest stage for an active pipeline. 
- The header should display the end time if the pipeline has finished running.
- The background colour of the header should be RED for failure, BLUE for ongoing and GREEN for successful builds.
- The back button should take you back to your build page.

### Comments:
